base/features/groovy/feature.yml

```diff
@@ -8,6 +8,4 @@ dependencies:
   - scope: compileOnly
     coords: io.micronaut:inject-groovy
   - scope: compile
-    coords: io.micronaut:runtime-groovy
-  - scope: testCompile
```

Comment:
What is the reasoning behind removing the dependency here? It's reasonable to assume someone with a Groovy project will want to write test classes in Groovy

Reply:
I had removed it to cut down a duplicate dependency between the `groovy` and `spock` features. It's back there now.